lib/Drush/CommandFiles/ExampleCommandFile.php

```diff
@@ -46,9 +48,9 @@ public function exampleTable($options = ['format' => 'table', 'fields' => ''])
      * @option $french Add a row with French numbers.
      * @usage example:formatters --french
      */
-    public function alterFormatters($result, array $args, AnnotationData $annotationData)
+    public function alterFormatters($result, CommandData $commandData)
     {
-        if ($args['options']['french']) {
+        if ($commandData->input()->getOption('french')) {
```

Comment:
OK thanks. There may be no need to change the name then ... Do you think that --druplicon should write to Output object instead of drush_print(). --druplicon is a silly edge case so maybe we just dont worry about it.